### ISSUE TITLE
Compatibility with Ears mod

### ISSUE THREAD

**Issue by notLily72** [enjarai/show-me-your-skin#24]
The mod: https://modrinth.com/mod/ears

When you turn off the chestplate armor piece with the ears mod installed, the chest setting of ears does not stop rendering. I have talked to unascribed, inorder for the rendering of the chest armor to be disabled the ears api should be used to tell ears whether to stop rendering the extra bit of armor. 

https://github.com/unascribed/Ears#using-the-api

How it currently looks: 
![image](https://user-images.githubusercontent.com/86197720/183543687-b5c3c88c-0629-4845-b2a5-d401bc1da8da.png)

How it should look
![image](https://user-images.githubusercontent.com/86197720/183543711-274e7702-90eb-4014-82d6-8c3788652514.png)

**Comment by enjarai** [comment]
I don't mind adding compat for this, but how would that work with transparency? The API does not seem to have support for that.

**Comment by notLily72** [comment]
yeah it appears that the api doesn't support it. Would it be possible to have it disable the armor piece completely once transparency is set to 0%?

**Comment by enjarai** [comment]
I'm trying to replicate this, but can't figure out how to make the chest show up using the ears mod. It's not an option on the skin configurator page, and I can't find any other documentation.

**Comment by notLily72** [comment]
It is currently a hidden option, if you would like I could upload my skin file with it?

**Comment by enjarai** [comment]
That'd work yes.

**Comment by notLily72** [comment]
![f](https://user-images.githubusercontent.com/86197720/184525717-6daa9f63-692c-4afa-a870-3615b1442721.png)

**Comment by chitinlink** [comment]
It's been a bit, is this looking like it'll be possible?

**Comment by enjarai** [comment]
I've been looking into it, but the ears codebase is extremely complex due to it being available for almost every minecraft version out there. 

When I'm implementing compat, I want it to support transparency as well. That usually requires a mixin into the other mod, which for most mods isn't a big problem. But due to the complexity of ears, I haven't been able to find the right place to inject my own stuff.

Currently, I don't have the free time to browse and comprehend all that code to find the specific spot where I need to change something, so I'll have to put this on hold until I do.

If you really want this compat, you could try contacting the dev of ears about implementing transparency support into the api.

**Comment by enjarai** [comment]
Actually, I contacted the dev myself just now, and they plan to add support for transparency to the api. Once that's available, I'll add compat.
